documentation/faq/FAQ.adoc

```diff
@@ -100,12 +100,37 @@ The different servers are listed as a comma separated string.
 Localhost is implicitly always in the white list.
 If you delete the line again then no whitelisting is applied and all servers are reachable.
 
+It is possible to specify wildcard (*) in the host and port part of the URL. Some examples:
+
```

Comment:
In my opinion this should be much shorter in the FAQ, just say that we support this and give 1-2 small examples, but this full table should be part of the OWS guide. In the "Server Whitelists" sections there is already a <TODO: describe the wildcards supported for whitelist entries introduced with OWS 1.2.0>